src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,30 +31,37 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              {`Todo #${selectedTodo?.id}`}
             </div>
 
             {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
             <button
               type="button"
               className="delete"
               data-cy="modal-close"
+              onClick={() => setSelectedTodo(null as unknown as Todo)}
```

Comment:
better to avoid assertions in TS

Reply:
I even don't know what is it 😅 
but with that 'unknown' linter showed mistake, so mentor offered this syntax

Reply:
we changed the type of that function, so you can remove assertions(as) here
<img width="455" alt="image" src="https://github.com/mate-academy/react_dynamic-list-of-todos/assets/70220667/e5f451fd-8f81-40e7-8cda-f8a116c68d02">
```suggestion
              onClick={() => setSelectedTodo(null)}
```